server/src/controllers/food.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from "express";
+import FoodService from "../services/food";
+
+export async function create(request: Request, response: Response) {
+  try {
+    const food = request.body.food;
+    const createdFood = await FoodService.create(food);
+    response.status(201);
+    response.json({ food: createdFood });
+  } catch (error) {
+    response.status(400);
+    response.json(error);
+  }
+}
+
+export async function createMany(request: Request, response: Response) {
+  try {
+    const foods = request.body.foods;
+    const createdFoods = await FoodService.createMany(foods);
+    response.status(201);
+    response.json({ foods: createdFoods });
```

Comment:
**Wont be needing this**
When returning an array of items, it would be helpful to also show how many items there are.
Here's what you currently have, which looks like:
`{ foods: [...] }`
A better response body would look like:
`{ count: 10, foods: [...] }`